tests/helpers.py

```diff
@@ -65,7 +65,7 @@ def extract(param_name: str, fit_bytes: bytes) -> typing.List[typing.Union[int,
         assert msg
         if msg.topic == callbacks_writer_pb2.WriterMessage.Topic.Value("SAMPLE"):
             for value_wrapped in msg.feature:
-                if param_name == value_wrapped.name:
+                if param_name == value_wrapped.name.decode():
```

Comment:
This is utf-8 by default?

Reply:
Yes, I believe utf-8 has been the default for some time. https://docs.python.org/3/library/stdtypes.html?highlight=decode#bytes.decode Is that what you mean?

Reply:
Yes